gson/src/test/java/com/google/gson/functional/PrintFormattingTest.java

```diff
@@ -47,7 +47,7 @@ protected void setUp() throws Exception {
 
   @SuppressWarnings({"unchecked", "rawtypes"})
   public void testCompactFormattingLeavesNoWhiteSpace() {
-    List list = new ArrayList();
+    List<?> list = new ArrayList<>();
     list.add(new BagOfPrimitives());
     list.add(new Nested());
     list.add(new PrimitiveArray());
```

Comment:
Could you please also replace the other raw type and remove the `@SuppressWarnings` annotation?
```java

  public void testCompactFormattingLeavesNoWhiteSpace() {
    List<Object> list = new ArrayList<>();
    list.add(new BagOfPrimitives());
    list.add(new Nested());
    list.add(new PrimitiveArray());
    list.add(new ClassWithTransientFields<>());
```